cnv_users_guide/cnv_users_guide.adoc

```diff
@@ -54,6 +54,10 @@ include::modules/cnv_openshift_commands.adoc[leveloffset=+2]
 include::modules/cnv_virtctl_commands.adoc[leveloffset=+2]
 include::modules/cnv_ensure_correct_project.adoc[leveloffset=+2]
 
+[[cnv_userguide_configuring_networking]]
+=== Configuring networking for virtual machines
```

Comment:
Do you need headings in both the assembly and the module?

Reply:
I wasn't sure where it should go in the overall assembly, so I created a separate networking section. It's not necessarily permanent, though, because we're planning to restructure the guides for CNV 2.0.